spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -551,7 +551,7 @@ function sendPacket(
   data: bytes) {
     channel = provableStore.get(channelPath(sourcePort, sourceChannel))
 
-    // check that the channel & connection are open to send packets; 
+    // check that the channel is not closed to send packets; 
     // note: optimistic sends are permitted once the handshake has started
```

Comment:
IMO, this comment is also misleading. If the handshake only did INIT, then optimistic sends do not work.

Reply:
I removed the comment.